magento/magento2
# Issue/37591 cart price rule incorect descount percent

### Description Cart price rules - Incorrect discount percent calculation when applying 2 or more sales rules by percent


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/magento2/issues/37591
2. Jira issue https://jira.corp.adobe.com/browse/AC-9120

### Manual testing scenarios (*)

1. Create 2 sales rules by percent
  1.1. Create 50% percent discount rule for the tested product item (set **Discard subsequent rules** to **No** value - need to apply few sales rules)
  1.2 Create 50% percent discount rule for the tested product item by using coupon code
2. As a result total discount should be 75%

### Questions or comments


### Contribution checklist (*)
